.gitignore

```diff
@@ -1,3 +1,4 @@
+temp.js
```

Comment:
Not sure what temp.js is, I was talking about plugin.zip to be added to gitignore